tests/helper/helper_run.go

```diff
@@ -35,6 +35,15 @@ func CmdShouldPass(program string, args ...string) string {
 	return string(session.Wait().Out.Contents())
 }
 
+// CmdShouldPassIncludeErrStream returns stdout and stderr if command succeeds
+func CmdShouldPassIncludeErrStream(program string, args ...string) (string, string) {
```

Comment:
The only difference between this function and `CmdShouldPass` seems like this one returns `stderr` as well. Why not modify `CmdShouldPass` instead? 

One reason for not doing so could be it would require modification at tonnes of places in this very PR. But keeping a separate function would mean that someone might forget that something like this exists.

If there's no other reason than having to modify `CmdShouldPass` at many places, I'd request we modify `CmdShouldPass` instead of having `CmdShouldPassIncludeErrStream`.

Reply:
currently - we cannot solve this problem other then rewriting the whole helper